modules/opensearch/security_groups.tf

```
@@ -49,3 +49,27 @@ resource "aws_security_group_rule" "snapshot_lambda_ingress" {
   security_group_id        = data.aws_security_group.es.id
   source_security_group_id = data.aws_security_group.snapshot_lambda.id
 }
+
+resource "aws_security_group_rule" "allow_secret_rotation_lambda_ingress_from_endpoint" {
+  description = "Allow secrets rotation lambda out to VPCE"
```

```suggestion
  description = "Allow secrets rotation lambda into to VPCE"
```

Sorry for nit picking 😅